### ISSUE TITLE
Pinch-Zoom allows adjusting y range, even when it should be fixed

### ISSUE THREAD

**Issue by b3nn0** [dbuezas/lovelace-plotly-graph-card#174]
```
type: custom:plotly-graph
layout:
  height: 400
entities:
  - entity: sensor.wolf_cgb_2_aussentemperatur
  - entity: sensor.dachboden_temperatur
  - entity: sensor.bme680_temperature
    name: Bürotemperatur
  - entity: sensor.wohnzimmertemperatur
defaults:
  entity:
    show_value: true
    line:
      shape: linear
  yaxes:
    fixedrange: true
hours_to_show: 48
title: Außentemperatur

```

However, when using the new pinch-zoom (which is awesome btw), I can still re-scale the Y axes, despited having it as `fixedrange: true`.
After the pinch-zoom gesture, it will be fixed again and I can only see the whole graph by hitting reset, or pinch-zooming out again.

I'd expect that pinch-zoom only zooms the X axis in this case, and leaves Y unchanged.

**Comment by dbuezas** [comment]
Oh, right, I didn't think about that. I'll fix it in the next release. 

Happy to hear you like it, I'm really enjoying it myself, particularly the tap tap drag gesture. I don't understand why plotly itself doesn't support it to be honest.

**Comment by dbuezas** [comment]
Closed by the release of v3.0.0
